src/Keboola/GoogleAnalyticsExtractor/Extractor/Extractor.php

```diff
@@ -171,6 +171,26 @@ public function getSampleReport($query)
         ];
     }
 
+    public function getJsonSampleReport($query)
```

Comment:
Len drobnost, ze by mi davalo vacsi zmysel, keby sa to volalo "getSampleReportJson". Teda "Json" by bola pripona k povodnemu nazvu.